README.md

```diff
@@ -10,7 +10,7 @@ Building
 make all
 ```
 
-The project uses essentially a copy and paste of the Makefile from the [regular input plugin for mupen64plus](https://github.com/mupen64plus/mupen64plus-input-sdl) With the dynamic config lib and SDL dependencies removed.
+The project uses essentially a copy and paste of the Makefile from the [regular input plugin for mupen64plus](https://github.com/mupen64plus/mupen64plus-input-sdl) With the dynamic config lib and SDL dependencies removed. It includes flags to link libjson-c, which it assumes has been installed in /usr/local/lib and /usr/local/include (you can get this library [here](https://github.com/json-c/json-c)).
```

Comment:
Looks like to get the headers, we need the `libjson-c-dev` package as well. I cleaned my custom json-c build, adjusted the make file and rebuilt the input library against the packaged version and it seemed to work. I'll update the README to include this info.